<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_lidar_filter_params</class>
 <widget class="QWidget" name="ihm_lidar_filter_params">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="params_list"/>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="param_value">
     <property name="minimum">
      <double>-100000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pb_apply">
     <property name="text">
      <string>apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
